web_src/js/features/common-issue.js

```diff
@@ -2,14 +2,33 @@ import $ from 'jquery';
 import {updateIssuesMeta} from './repo-issue.js';
 
 export function initCommonIssue() {
-  $('.issue-checkbox').on('click', () => {
-    const numChecked = $('.issue-checkbox').children('input:checked').length;
-    if (numChecked > 0) {
+  $('.issue-checkbox,.issue-checkbox-all').on('click', (e) => {
```

Comment:
Should this be in one click event? Seems better for code readability to separate it.